src/resources/controllers/resource-subscriber/test.tsx

```diff
@@ -98,26 +96,24 @@ describe('<ResourceSubscriber />', () => {
       },
     });
 
-    const Component = () => (
+    render(
       <ResourceSubscriber resource={mockResource}>
         {({ data, loading }) => {
           if (loading) {
-            return <div id="loading" />;
+            return <div data-testid="loading" />;
```

Comment:
nit: probably better to do 
```suggestion
            return <div>loading</div>;
```
and query by text 

not a blocker though